basics/concepts/time-boundary.md

```diff
@@ -0,0 +1,71 @@
+---
+description: In this Apache Pinot concepts guide, we'll learn about time boundaries in hybrid tables.
+---
+
+# Time Boundary
+
+In this guide we'll learn about time boundaries in hybrid tables.
+Hybrid tables are when we have offline and real-time tables with the same name.
```

Comment:
```suggestion
Learn about time boundaries in hybrid tables.
Hybrid tables are when we have offline and real-time tables with the same name.
```

Reply:
Hybrid tables occur when you create an offline and real-time table with the same name?